quran/models.py

```diff
@@ -1,19 +1,29 @@
 '''Models describing abstractions used at Quran app'''
-from django.db import migrations, models
+from django.db import models
 
 
 class Surah(models.Model):
-    '''General information about Surahs'''
+
+    ''' General information about Surah '''
+
     number = models.PositiveIntegerField(primary_key=True)
     name = models.CharField(max_length=20)
     total_ayahs = models.PositiveIntegerField()
 
+    def __str__(self):
```

Comment:
Depends on where are you going to use this function, but I would prefer to see a surah number in the brackets too.